lib/commit.tcl

```diff
@@ -208,30 +236,13 @@ You must stage at least 1 file before you can commit.
 	# -- A message is required.
 	#
 	set msg [$ui_comm get 1.0 end]
-	# Strip trailing whitespace
-	regsub -all -line {[ \t\r]+$} $msg {} msg
-	# Strip comment lines
-	global comment_string
-	set cmt_rx [strcat {(^|\n)} [regsub -all {\W} $comment_string {\\&}] {[^\n]*}]
-	regsub -all $cmt_rx $msg {\1} msg
-	# Strip leading and trailing empty lines (puts adds one \n)
-	set msg [string trim $msg \n]
-	# Compress consecutive empty lines
-	regsub -all {\n{3,}} $msg "\n\n" msg
-	if {$msg eq {}} {
-		error_popup [mc "Please supply a commit message.
-
-A good commit message has the following format:
-
-- First line: Describe in one sentence what you did.
-- Second line: Blank
-- Remaining lines: Describe why this change is good.
-"]
-		unlock_index
+	# Check if message would be empty after processing
```

Comment:
the commit message doesn't make it clear whether that extra check is also consistent with git.
if it isn't, then i'd be wary of adding it here.